pkg/config/default_windows.go

```diff
@@ -42,11 +42,17 @@ func getLibpodTmpDir() string {
 }
 
 // getDefaultMachineVolumes returns default mounted volumes (possibly with env vars, which will be expanded)
+// It is executed only if the machine provider is Hyper-V and it mimics WSL
+// behavior where the host %SystemDrive% (e.g. C:\) is automatically mounted
+// in the guest under /mnt/ (e.g. /mnt/c/)
```

Comment:
The comment doesn't quite match the code, we mount the drive where the users homedir is located.
I don't know if there is a way for a homedir to be on a different disk compared to  C: (%SystemDrive%) in practise (at least super unlikely) so not really that important I guess but maybe we should actually lookup the `SystemDrive` env?

Reply:
Technically the drive of %USERPROFILE% can be different from %SYSTEMDRIVE%. But I still think we should mount %USERPROFILE% drive rather than %SYSTEMDRIVE% as user files are typically there. I have updated the comment.